ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/Footer.kt

```diff
@@ -83,7 +83,7 @@ private fun Footer(
     ) {
         val color = colors.text1
 
-        if (mode == PaywallMode.FOOTER_CONDENSED) {
+        if (mode == PaywallMode.FOOTER_CONDENSED && allPlansTapped != null) {
```

Comment:
Since template 1 doesn't support the condensed footer version (it's already a single option to begin with), I made the parameter nullable, and use that to detect whether to show the button or not. This is the easiest way I think to not show the condensed button in templates where it's not supported.

Reply:
That makes sense. Looking at iOS I did the same thing.